example/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:latest
+
+RUN mkdir -p /lib64 && ln -sf /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
+RUN apk add --no-cache py3-pip
+RUN pip install requests
```

Comment:
FWIW, requests is packaged in alpine. Replacing these two lines with  `apk add --no-cache py3-requests` saves about 20 MB on my system. But so does passing `--no-cache` to pip.

You can also `FROM python:alpine` and remove the `apk` invocation entirely. I suggest that as more natural for a python app...especially if there were a requirements.txt. :)

Reply:
Good to know! Fixed in 8710b38.